cypress/e2e/hermioneAccount.spec.js

```diff
@@ -1,11 +1,98 @@
 /// <reference types='cypress' />
+import { faker } from '@faker-js/faker';
 
 describe('Bank app', () => {
-  before(() => {
-    cy.visit('/');
+  beforeEach(() => {
+    cy.visit('');
```

Comment:
The URL in the `cy.visit('');` command is empty. You should provide the URL of the page you want to visit.